test/Swashbuckle.AspNetCore.SwaggerGen.Test/SchemaGenerator/JsonSerializerSchemaGeneratorTests.cs

```diff
@@ -15,6 +15,7 @@
 using Xunit;
 using Swashbuckle.AspNetCore.TestSupport;
 using Microsoft.OpenApi.Any;
+using System.ComponentModel.DataAnnotations;
```

Comment:
Nit: could you sort the usings please? They should be System first then the rest (#2790).